<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>666</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>666</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FIDMAA</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="loadJPEGButton">
         <property name="text">
          <string>&amp;Load image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Midline point - click at the chin to set:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>x:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="xValue">
           <property name="maximum">
            <number>480</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>y:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="yValue">
           <property name="maximum">
            <number>640</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>angle (skew):</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="angleValue">
           <property name="suffix">
            <string> deg</string>
           </property>
           <property name="prefix">
            <string/>
           </property>
           <property name="minimum">
            <number>45</number>
           </property>
           <property name="maximum">
            <number>135</number>
           </property>
           <property name="value">
            <number>90</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSlider" name="angleSlider">
         <property name="minimum">
          <number>45</number>
         </property>
         <property name="maximum">
          <number>135</number>
         </property>
         <property name="value">
          <number>90</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Averaging area (width):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="areaValue">
         <property name="suffix">
          <string> pixels</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="angleSlider_2">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Depth cutoff:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="depthCutoffValue">
         <property name="suffix">
          <string> pixels</string>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>127</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>127</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="dataOutputEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0" rowspan="2">
      <widget class="QClickableLabel" name="imageLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>480</width>
         <height>640</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>480</width>
         <height>640</height>
        </size>
       </property>
       <property name="text">
        <string>Image</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QClickableLabel" name="chartLabel">
     <property name="minimumSize">
      <size>
       <width>255</width>
       <height>640</height>
      </size>
     </property>
     <property name="text">
      <string>Chart</string>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionShow_chart">
   <property name="text">
    <string>Show chart</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QClickableLabel</class>
   <extends>QLabel</extends>
   <header>qclickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>angleSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>angleValue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>627</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>angleSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>areaValue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>558</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>578</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>angleValue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>angleSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>624</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>633</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>areaValue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>angleSlider_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>598</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>610</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>depthCutoffValue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>545</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>depthCutoffValue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>636</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>636</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
